src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPService.kt

```diff
@@ -132,6 +134,7 @@ class CSIPService(
       ?: log.info("CSIP deletion request for: $csipId ignored. CSIP does not exist")
   }
 
+  @Audit
```

Comment:
Are you sure we still need that annotation ? I think it was originally needed for the module_name thing but is now done automatically

Reply:
I wasn't sure tbh - I will remove (I think I copied one of the other places that had it set) - I was gonna speak to you about it but then forgot. - will remove in next PR